t/test.lua

```diff
@@ -196,6 +196,13 @@ function issue_120(request)
   return request
 end
 
+function skipping_test(...)
+  for k, v in pairs({...}) do
+    print(k, "->", require 'json'.encode(v))
+  end
+    return 201, "OPA", { ['X-From-Tarantool'] = 'Oh yes' }
```

Comment:
This is a "POST". Why are you returning 201?

Reply:
Now it is 200.